css/main.css

```diff
@@ -207,3 +202,74 @@ hr {
   border-bottom: 1px solid var(--neutral);
   margin: 12px auto;
 }
+
+/* contact form start here */
+.contact-form-container {
+  width: 100%;
+  background: url('../images/contact-form.png'), var(--logo-purple);
+  background-repeat: no-repeat;
+
+  /* background-attachment: fixed; */
+  background-size: 100% 100%;
+  border-top-left-radius: 50px;
+  height: auto;
+  padding: 60px 0;
+}
+
+.contact-form-container .contact-form {
+  max-width: 447px;
+  text-align: center;
+  margin: auto;
+  padding: 50px 18px;
+}
+
+.contact-form h2 {
+  color: #fff;
+  margin: 30px 0 12px;
+}
+
+.contact-form p {
+  font-weight: 400;
```

Comment:
I think the default `font-weight` is 400 i.e. regular, so this is redundant, and it's also repeated in other parts of your `css`

Reply:
Adjusted sir

Reply:
All is been resolved sir